sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,38 +1,37 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (20/11/10)
```

Comment:
```suggestion
## 1.0.0-beta.3 (2020-11-10)
```

Reply:
@ahsonkhan Good catch on the date format